web/core/components/analytics-v2/insight-card.tsx

```
@@ -0,0 +1,46 @@
+  data?: IAnalyticsResponseFieldsV2;
+  label: string;
+  isLoading?: boolean;
+}
+
+const InsightCard = (props: InsightCardProps) => {
+  const { data, label, isLoading } = props;
+  const { count, filter_count } = data || {};
+  const percentage = useMemo(() => {
+    if (count != null && filter_count != null) {
+      const result = ((count - filter_count) / count) * 100;
+      return isFinite(result) ? result : null;
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)